tensor2tensor/rl/collect.py

```diff
@@ -17,44 +17,99 @@
 from __future__ import division
 from __future__ import print_function
 
+
+import copy
 import tensorflow as tf
+from tensor2tensor.rl.envs import utils
+from tensor2tensor.rl.envs.tf_atari_wrappers import WrapperBase
+from tensor2tensor.rl.envs.utils import get_policy
 
 
-def define_collect(policy_factory, batch_env, hparams,
-                   eval_phase, policy_to_actions_lambda=None,
-                   scope="", preprocess=None, on_simulated=False):
-  """Collect trajectories."""
-  eval_phase = tf.convert_to_tensor(eval_phase)
-  on_simulated = tf.convert_to_tensor(on_simulated)
+def _rollout_metadata(batch_env):
   batch_env_shape = batch_env.observ.get_shape().as_list()
-  if preprocess is not None:
-    batch_env_shape = preprocess[1]
-  memory_shape = [hparams.epoch_length] + [batch_env_shape[0]]
-  memories_shapes_and_types = [
-      # observation
-      (memory_shape + batch_env_shape[1:], tf.float32),
-      (memory_shape, tf.float32),      # reward
-      (memory_shape, tf.bool),         # done
-      # action
-      (memory_shape + batch_env.action_shape, batch_env.action_dtype),
-      (memory_shape, tf.float32),      # pdf
-      (memory_shape, tf.float32),      # value function
+  batch_size =  [batch_env_shape[0]]
+  shapes_types_names = [
+    # observation
+    (batch_size + batch_env_shape[1:], tf.float32, "observation"),
```

Comment:
This has wrong indentation, can you make sure that linter is passing in travis?

Reply:
I do not know how to check it in travis. I did a check against tensor2tensor.pylintrc.